packages-content-model/roosterjs-content-model-core/lib/override/pasteGeneralProcessor.ts

```diff
@@ -1,12 +1,22 @@
+import { AllowedTags, createSanitizedElement, DisallowedTags } from '../utils/sanitizeElement';
 import { moveChildNodes } from 'roosterjs-content-model-dom';
-import {
-    AllowedTags,
-    createSanitizedElement,
-    DisallowedTags,
-    removeDisplayFlex,
-    removeStyle,
-} from '../utils/sanitizeElement';
-import type { DomToModelOptionForPaste, ElementProcessor } from 'roosterjs-content-model-types';
+import type {
+    DomToModelOptionForPaste,
+    ElementProcessor,
+    ValueSanitizer,
+} from 'roosterjs-content-model-types';
+
+/**
+ * @internal Export for test only
+ */
+export const removeDisplayFlex: ValueSanitizer = value => {
```

Comment:
Why do we remove display:flex? Does remove it can break the formatting of the copied fragment?

Reply:
It was from this change: https://github.com/microsoft/roosterjs/pull/2031

When paste with display:flex, it may cause some unexpected editing behavior. For example, with horizontal flex display, moving cursor Up/Down will cause cursor move to the next container (on right side) rather than the one below it. So we decide to remove this style.

Although there is still some other debate about this, for now I'll still keep this behavior, until we have a better solution.